eng/build.ps1

```diff
@@ -366,14 +383,23 @@ function TestUsingRunTests() {
   if ($testCoreClr) {
     $args += " --tfm net5.0"
     $args += " --tfm netcoreapp3.1"
-    $args += " --include '\.UnitTests'"
     $args += " --timeout 90"
+    if ($testCompilerOnly) {
+      $args += GetCompilerTestAssembliesIncludePaths
+    } else {
+      $args += " --include '\.UnitTests'"
+    }
   }
-  elseif ($testDesktop -or $testIOperation) {
+  elseif ($testDesktop -or ($testIOperation -and -not $testCoreClr)) {
```

Comment:
this additional logic looks like it's meant to make it so you only need to pass the single `.\Test.cmd -testIOperation` flag to run ioperation tests locally. Is that right?

Reply:
Correct, that's the behavior we have today.